core/controllers/acl_decorators_test.py

```diff
@@ -2144,26 +2148,30 @@ def setUp(self):
 
     def test_cannot_access_non_existent_story(self):
         with self.swap(self, 'testapp', self.mock_testapp):
-            self.get_json('/mock_story/story_id', expected_status_int=404)
+            self.get_json(
+                '/mock_story/topic/story_id', expected_status_int=404)
 
     def test_cannot_access_story_when_topic_is_not_published(self):
         with self.swap(self, 'testapp', self.mock_testapp):
             self.get_json(
-                '/mock_story/%s' % self.story_id, expected_status_int=404)
+                '/mock_story/topic/%s' % self.story_id,
+                expected_status_int=404)
 
     def test_cannot_access_story_when_story_is_not_published(self):
         topic_services.publish_topic(self.topic_id, self.admin_id)
         with self.swap(self, 'testapp', self.mock_testapp):
             self.get_json(
-                '/mock_story/%s' % self.story_id, expected_status_int=404)
+                '/mock_story/topic/%s' % self.story_id,
+                expected_status_int=404)
 
     def test_can_access_story_when_story_and_topic_are_published(self):
         topic_services.publish_topic(self.topic_id, self.admin_id)
         topic_services.publish_story(
             self.topic_id, self.story_id, self.admin_id)
         with self.swap(self, 'testapp', self.mock_testapp):
             self.get_json(
-                '/mock_story/%s' % self.story_id, expected_status_int=200)
+                '/mock_story/topic/%s' % self.story_id,
+                expected_status_int=200)
 
 
```

Comment:
You should have way more tests than this, given the amount of new code that has been added to acl_decorators.py.

Reply:
Done.